recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/guest_domf

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+#
+# xencommons    Script to start DomF guest
+#
+# Author:       Andrii Anisov <andrii_anisov@epam.com>
```

Comment:
I'm not sure, that Andrii had wrote this  file.

Reply:
Also, speaking more generally, I would like to have one generic script that can be instantiated with domain name, rather then bunch of identical scripts.

Reply:
I'm not sure it is feasible

Reply:
once we have decided what is the domain boot order we may do this
at the moment I would prefer it as is